src/replicache.ts

```diff
@@ -527,6 +583,16 @@ export class Replicache<MD extends MutatorDefs = {}> {
     return this._licenseCheckPromise;
   }
 
+  /**
+   * Indicates whether the license passed to Replicache is active. An
+   * active license is one that we are sending licensing active pings
+   * for. We do not send pings when licensing is disabled (license key is
+   * undefined) or for the TEST_LICENSE_KEY.
+   */
+  get licenseActive(): Promise<boolean> {
```

Comment:
What is the use case for this property?

Reply:
I'm worried about this becoming public API, which we would not want because we'd then be stuck supporting it even though it only exists for testing. Maybe there is some build mechanism I'm not aware of that prevents this and so my worry is invalid? Or maybe there's some way to mark the APi so it doesn't show up in docs as public but is still available on the class for testing?

Another way to control the asynchrony of the tests would be to use `sinon` to override the timers under test and force a number of ms to move forward and for timers to go off. Look for heartbeat.test.ts for example. I think this might be a win for complexity too as it would mean not having to wire these resolvers through the call stack and not having to track the extra state.

Reply:
ffwding timers does not solve the same problem: it can trigger timers but it doesn't tell us when an already-initiated call has completed. 

i understand the concern tho about putting it in the public api. i think the cleanest thing might be to make the accessor private and then in tests that need it add a mixin to replicache that promotes the call to public. is that a thing we do in javascript? if not then other options:
- an accessor for the license key could be a natural part of the api. if we had such an accessor we could gate it on the underlying license check and active ping calls, maybe including those bits along with the key in the return? 
- it could be hidden from generated jsdoc using `@ignore`
- make the accessor protected, inherit from repliache in the test that needs it, and promote the method to public for the test